sphinx/source/docs/user_guide/extensions.rst

```diff
@@ -107,7 +107,7 @@ the final BokehJS scripts.
         // slider value.
         super.render()
 
-        this.el.appendChild(div({
+        this.shadow_el.appendChild(div({
```

Comment:
If there is nothing we can do to automatically ease this transition, we need to at least describe necessary changes in the migration guide

Reply:
I'm not sure at this stage (as I said earlier, this is a transitional PR) if this API will be kept as-is, especially given the extent of changes to come. When the APIs settle, then we can think about migration guide.